source/rhoas/src/main/java/com/openshift/cloud/beans/KafkaApiClient.java

```diff
@@ -1,14 +1,14 @@
 package com.openshift.cloud.beans;
 
-import com.openshift.cloud.ApiClient;
-import com.openshift.cloud.ApiException;
-import com.openshift.cloud.Configuration;
-import com.openshift.cloud.api.DefaultApi;
+import com.openshift.cloud.api.ApiClient;
+import com.openshift.cloud.api.ApiException;
```

Comment:
our of curiosity - why this is not namespaced?

Reply:
They're common classes that are shared by both implementations. The ApiClient handles Http, suth, etc and ApiException is an execption.

Reply:
Ok.. So this is quite well done. I see duplication in golang so need to figure out if there is the same option for golang genator to have them separately.

CC @craicoverflow